tests/executors/test_celery_executor.py

```diff
@@ -469,3 +470,23 @@ def test_should_support_base_backend(self):
                 )
 
         assert result == {'123': ('SUCCESS', None), '456': ("PENDING", None)}
+
+    @pytest.mark.integration("redis")
+    @pytest.mark.integration("rabbitmq")
+    @pytest.mark.backend("mysql", "postgres")
+    def test_should_support_base_backend_from_try_adopt_task_instances(self):
```

Comment:
Do you need to set logging to debug for this test to cover the new code?

Reply:
Good call.

I refactored the tests for the possible iterations of `get_many` to set the log mode to `DEBUG` and updated the debug statement to account for being a map object vs being a list object.